src/tick_math.rs

```diff
@@ -97,6 +97,11 @@ pub fn get_sqrt_ratio_at_tick(tick: i32) -> Result<U256, UniswapV3MathError> {
         })
 }
 
+const MAGIC_SQRT_10001: I256 = I256::from_raw(U256([11745905768312294533, 13863, 0, 0]));
+const MAGIC_TICK_LOW: I256 = I256::from_raw(U256([6552757943157144234, 184476617836266586, 0, 0]));
+const MAGIC_TICK_HIGH: I256 =
+    I256::from_raw(U256([4998474450511881007, 15793544031827761793, 0, 0]));
+
```

Comment:
Can you change the name to `SQRT_10001`,`TICK_LOW` and `TICK_HIGH`?

Also can you move these up with the rest of the constants?